### PR TITLE
Validate: print error as error instead info

### PR DESCRIPTION
**Pull request by ilappe**
related: https://jira-hq.paloaltonetworks.local/browse/CIAC-7116
## Description
currently some validation error printed as `INFO` instead `ERROR`

image

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@
 * Fixed an issue where **validate** always returned *XSIAM Dashboards* and *Correlation Rules* files as valid.
 * Added `GR107` validation to **validate** using the graph validations to check that no deprecated items are used by non-deprecated content.
 * Fixed an issue where the **modeling-rules test** command failed to get the existence of dataset in cases where the dataset takes more than 1 minute to get indexed.
+* Fixed an issue where errors in **validate** was printed to logs as`INFO`.
```
Comment (tkatzir) [inline CHANGELOG.md:13]:
```suggestion
* Fixed an issue where errors in **validate** were legged as `INFO`.
```

**Review by tkatzir — APPROVED** [review]
Good!
Approved, with a comment on the CHANGELOG.

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,6 +10,7 @@
 * Fixed an issue where **validate** always returned *XSIAM Dashboards* and *Correlation Rules* files as valid.
 * Added `GR107` validation to **validate** using the graph validations to check that no deprecated items are used by non-deprecated content.
 * Fixed an issue where the **modeling-rules test** command failed to get the existence of dataset in cases where the dataset takes more than 1 minute to get indexed.
+* Fixed an issue where errors in **validate** were legged as `INFO`.
```
Comment (dorschw) [inline CHANGELOG.md:13]:
```suggestion
* Fixed an issue where errors in **validate** were logged as`info`.
```

Comment (dorschw) [reply CHANGELOG.md:13]: @tkatzir :)

File: CHANGELOG.md [hunk 3]
```diff
@@ -10,6 +10,7 @@
 * Fixed an issue where **validate** always returned *XSIAM Dashboards* and *Correlation Rules* files as valid.
 * Added `GR107` validation to **validate** using the graph validations to check that no deprecated items are used by non-deprecated content.
 * Fixed an issue where the **modeling-rules test** command failed to get the existence of dataset in cases where the dataset takes more than 1 minute to get indexed.
+* Fixed an issue where errors in **validate** were logged as`info`.
```
Comment (dorschw) [inline CHANGELOG.md:13]:
```suggestion
* Fixed an issue where errors in **validate** were logged as `info`.
```

**Pull request merged**